bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -0,0 +1,334 @@
+channel-type.netatmo.rssi.description = Signal strength indicator.
+channel-type.netatmo.sd-card-status.label = SD Card State
+channel-type.netatmo.sd-card-status.description = State of the SD card
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_MISSING = Missing SD Card
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_INSERTED = SD Card inserted
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_FORMATTED = SD Card formated
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_WORKING = Working SD Card
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_DEFECTIVE = Defective SD Card
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_INCOMPATIBLE_SPEED = Incompatible SD Card speed
+channel-type.netatmo.sd-card-status.state.option.SD_CARD_INSUFFICIENT_SPACE = Insufficient SD Card space
+channel-type.netatmo.setpoint-duration.label = Setpoint Duration
+channel-type.netatmo.setpoint-duration.description = Default duration of manual setpoint changes.
+channel-type.netatmo.setpoint.label = Setpoint
+channel-type.netatmo.setpoint.description = Thermostat temperature setpoint.
+channel-type.netatmo.th-mode.label = Thermostat Mode
+channel-type.netatmo.th-mode.description = Chosen thermostat mode (home, frost guard, manual, max).
+channel-type.netatmo.th-mode.state.option.HOME = Home
+channel-type.netatmo.th-mode.state.option.FROST_GUARD = Frost Guard
+channel-type.netatmo.th-mode.state.option.MANUAL = Manual
+channel-type.netatmo.th-mode.state.option.MAX = Max
+channel-type.netatmo.timestamp-advanced.label = Timestamp
+channel-type.netatmo.timestamp-advanced.description = Moment when data was measured.
+channel-type.netatmo.timestamp.label = Timestamp
+channel-type.netatmo.timestamp.description = Moment when data was measured.
+channel-type.netatmo.trend.label = Trend
+channel-type.netatmo.trend.description = Evolution of the measure over time.
+channel-type.netatmo.trend.state.option.UP = Up
+channel-type.netatmo.trend.state.option.STABLE = Stable
+channel-type.netatmo.trend.state.option.DOWN = Down
+channel-type.netatmo.unknown-person-count.label = Unknown Persons Count
+channel-type.netatmo.unknown-person-count.description = Total number of unknown persons that are at home.
+channel-type.netatmo.unknown-person-picture.label = Unknown Person Snapshot
+channel-type.netatmo.unknown-person-picture.description = Snapshot of unknown person that is at home.
+channel-type.netatmo.video-status.label = Video Status
+channel-type.netatmo.video-status.description = Status of the video (recording, deleted or available).
+channel-type.netatmo.video-status.state.option.RECORDING = Recording
+channel-type.netatmo.video-status.state.option.DELETED = Deleted
+channel-type.netatmo.video-status.state.option.AVAILABLE = Available
+channel-type.netatmo.video-url.label = Video URL
+channel-type.netatmo.window-open.label = Window Status
+channel-type.netatmo.window-open.descrption = Windows of the room are opened.
```

Comment:
typo: description
Found by simply running the maven command to build this file.

Reply:
Corrected.